### PR TITLE
Add Route describer

### PR DESCRIPTION
**Pull request by spadgett**
Add object describer for routes.

kubernetes_describer_example

### CODE REVIEW

**Comment by spadgett** [comment]
@liggitt please review

File: bower.json [hunk 1]
```diff
@@ -24,7 +24,8 @@
     "dist/object-describer.js"
   ],
   "dependencies": {
-    "kubernetes-object-describer": "0.0.10"
+    "kubernetes-object-describer": "0.0.10",
+    "jquery": "~2.1.4"
```
Comment (liggitt) [inline bower.json]: can you put jquery in devDependencies, since we only require it for the index.html?

File: views/route.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    <dt>Service Name</dt>
+    <dd>{{resource.serviceName}}</dd>
+  </dl>
+  <div ng-if="resource.tls">
+    <h3>TLS</h3>
+    <dl class="dl-horizontal">
+      <dt>Termination</dt>
+      <dd>{{resource.tls.termination || 'None'}}</dd>
+      <dt>Certificate</dt>
+      <dd collapse-long-text value="{{resource.tls.certificate || 'None'}}"></dd>
```
Comment (liggitt) [inline views/route.html]: do these values have newlines? can we style them with `white-space:pre; overflow-x:auto;`? maybe `font-family:monospace; font-size: .9em;` or similar as well

Comment (liggitt) [inline views/route.html]: hmm, not sure that will work with `collapseLongText`... might need to take extra whitespace out of `_collapse-long-text.html` to allow that to work

**Comment by spadgett** [comment]
@liggitt I'm now masking the long certificate and private key values with `*****`.

File: views/route.html [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    <dt>Created</dt>
+    <dd>{{resource.metadata.creationTimestamp}}</dd>
+    <dt>Host</dt>
+    <dd>{{resource.host}}</dd>
+    <dt>Path</dt>
+    <dd>{{resource.path || 'None'}}</dd>
+    <dt>Service Name</dt>
+    <dd>{{resource.serviceName}}</dd>
+  </dl>
+  <div ng-if="resource.tls">
```
Comment (liggitt) [inline views/route.html]: add `&& resource.tls.termination`... from API docs (https://github.com/openshift/origin/blob/master/pkg/route/api/v1beta3/types.go#L75-L92): "Termination indicates termination type.  If termination type is not set, any termination config will be ignored"

File: views/route.html [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    <h3>TLS</h3>
+    <dl class="dl-horizontal">
+      <dt>Termination</dt>
+      <dd>{{resource.tls.termination || 'None'}}</dd>
```
Comment (liggitt) [inline views/route.html]: can assume we have a value here

File: views/route.html [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      <dt>Certificate</dt>
+      <dd>{{(resource.tls.certificate) ? '*****' : 'None'}}</dd>
+      <dt>Key</dt>
+      <dd>{{(resource.tls.key) ? '*****' : 'None'}}</dd>
+      <dt>CA Certificate</dt>
+      <dd>{{(resource.tls.caCertificate) ? '*****' : 'None'}}</dd>
+    </dl>
```
Comment (liggitt) [inline views/route.html:29]: add `destinationCACertificate`

**Comment by spadgett** [comment]
@liggitt updated for your comments

File: bower.json [hunk 6]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "openshift-object-describer",
-  "version": "0.0.7",
```
Comment (liggitt) [inline bower.json:3]: let's stop bumping bower versions in the same commits as improvements... I can bump them when I cut a release

**Comment by spadgett** [comment]
kubernetes_describer_example

**Pull request merged**